bin/cron/deploy_to_test

```diff
@@ -9,6 +9,7 @@ require 'cdo/only_one'
 
 TOPIC_DTT_IN_PROGRESS = 'no (robo-DTT in progress)'.freeze
 TOPIC_DTT_FAILED = 'robo-DTT failed'.freeze
+ALWAYS_DTT = ARGV[0] && ARGV[0] == 'always_dtt'
```

Comment:
Fine for now, but if we ever add a second argument I think it's worth loading `optparse` so we don't have to care about the order of flags.